code/datums/diseases/advance/advance.dm

```diff
@@ -424,4 +424,9 @@ var/list/advance_cures = 	list(
 		src << "<a href='?_src_=vars;Vars=\ref[D]'>[D.name] - [D.holder]</a>"
 */
 
+/datum/symptom/heal/proc/Heal(mob/living/M, datum/disease/advance/A)
+	var/get_damage = sqrt(20-"stage_rate")*(2+rand())

```

Comment:
I'm guessing "stage_rate" is a key for the ```properties``` associative list. You're not accessing that list though, you're just getting the square root of

```20 - "stage_rate"```

Which makes no sense since you're doing a minus operation to a string.